mms/log.py

```diff
@@ -99,6 +85,6 @@ def get_logger(name=None, level="NOTSET"):
     return logger
 
 
-def log_msg(*args, **kwargs):
+def log_msg(*args):
```

Comment:
Good catch. It might be needed though, if we are passing a set of key-value pairs. I will work on added such a log capability.